lxd/apparmor/instance_lxc.go

```diff
@@ -28,7 +28,9 @@ profile "{{ .name }}" flags=(attach_disconnected,mediate_deleted) {
 
   # Handle binfmt
   mount fstype=binfmt_misc -> /proc/sys/fs/binfmt_misc/,
-  deny /proc/sys/fs/binfmt_misc/{,**} rwklx,
+  {{- if not .kernel_binfmt }}
+    deny /proc/sys/fs/binfmt_misc/{,**} rwklx,
```

Comment:
Since it's akin to a preprocessor `if`, I wouldn't indent the body as it would stick out... cosmetic only OFC.